tests/test_indexutils.py

```diff
@@ -66,7 +82,8 @@ def test_analyze(self):
         self.assertEqual(' '.join(self.index_utils.analyze('rapid retrieval, space economy')),
                          'rapid retriev space economi')
         tokenizer = pyanalysis.get_lucene_analyzer(stemming=False)
-        self.assertEqual(' '.join(self.index_utils.analyze('retrieval', analyzer=tokenizer)), 'retrieval')
```

Comment:
Are these lines over 100 chars? Unnecessary changes?

Reply:
And below as well...

Reply:
Yea, these are 107 chars. I changed my pylint line length = 100.